### PR TITLE
Wrap with quotes to stop splitting

### PR DESCRIPTION
**Pull request by benjefferies**
### What is the context of this PR?
Generating schemas will fail if there are spaces in the path.

Depends on https://github.com/ONSdigital/eq-translations/pull/8

### How to review 
1. Clone repository into a directory with spaces in `git clone https://github.com/ONSdigital/eq-survey-runner.git eq\ survey\ runner`
1. Run  `./scripts/translate_schemas.sh`

### CODE REVIEW

File: scripts/run_app.sh [hunk 1]
```diff
@@ -24,7 +24,7 @@ fi
 
 # Output the current git revision
 if [ -z "$EQ_GIT_REF" ]; then
```
Comment (jonnyshaw89) [inline scripts/run_app.sh:20]: do we still need any of this? it was used to determine if the app had changed but no is only displayed on the status page?

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

File: scripts/run_app.sh (hunk 1 above)
Comment (collisdigital) [reply scripts/run_app.sh:20]: There was also discussion about it being somewhere in the log output at startup for Splunk, is it logged anywhere?

Comment (benjefferies) [reply scripts/run_app.sh:20]: Yeah it's logged out on start up

File: scripts/run_app.sh [hunk 2]
```diff
@@ -40,12 +40,12 @@ fi
 echo "Environment variables in use:"
 env | grep EQ_
 
-$DIR/build.sh
+"${DIR}"/build.sh
 
-url="`python token_generator.py`"
+url=$(python token_generator.py)
```
Comment (jonnyshaw89) [inline scripts/run_app.sh]: Can we get rid of this yet? no one likes it :)

Comment (benjefferies) [reply scripts/run_app.sh]: I'm happy to get rid of it. Everyone goes to the dev page now surely?

Comment (jonnyshaw89) [reply scripts/run_app.sh]: Yeh, probably just launch the dev page

Comment (benjefferies) [reply scripts/run_app.sh]: Done

**Review by jonnyshaw89 — APPROVED** [review]

**Review by ajmaddaford — APPROVED** [review]

**Pull request merged**